src/TreePiece.h

```diff
@@ -487,7 +487,7 @@ void TreePiece<Data>::perturb (Real timestep, bool if_flush) {
       Vector3D<Real> old_position = particle.position;
       particle.perturb(timestep, leaf->getForce(i), readers.ckLocalBranch()->universe.box);
       //CkPrintf("magitude of displacement = %lf\n", (old_position - leaf->particles[i].position).length());
-      //CkPrintf("total centroid is (%lf, %lf, %lf)\n", global_root->data.getCentroid().x, global_root->data.getCentroid().y, global_root->data.getCentroid().z);
+      //CkPrintf("total centroid is (%lf, %lf, %lf)\n", global_root->data.centroid.x, global_root->data.centroid.y, global_root->data.centroid.z);
```

Comment:
Can we simply remove these debug print statements or put them under some kind of verbosity control? Perhaps an #ifdef?

Reply:
More of a general question than anything directly pertaining to this commit.